engine/Shopware/Kernel.php

```diff
@@ -741,6 +741,7 @@ protected function getKernelParameters()
             'kernel.debug' => $this->debug,
             'kernel.name' => $this->name,
             'kernel.cache_dir' => $this->getCacheDir(),
+            'kernel.cache_dir.product_export' => $this->getCacheDir().'/productexport',
```

Comment:
the places where you replaced the code with your new DI parameter looked like this `$cacheDir .= '/productexport/';`(or similar :wink: )

so I guess
```suggestion
            'kernel.cache_dir.product_export' => $this->getCacheDir().'/productexport/',
```
a slash is missing at the end? :thinking:

Reply:
Yes good catch, bad copy and past from me. I developed it in our local docker setup and not directly in this repo 👼🏻